github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1293,3 +1293,62 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 7,15,23 * * *"
+  decorate: true
+  cluster: prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-bazel-cache: "true"
+    preset-bazel-unnested: "true"
+    group: kubevirtci
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: main
+    work_dir: true
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+        env:
```

Comment:
Can it be that we have to add

```yaml
- name: KUBEVIRT_PROVIDER_EXTRA_ARGS
  value: "--prometheus-port 30007"
```

to give prometheus the fixed reachable port which we expect at https://github.com/kubevirt/project-infra/blob/15807c5f709d63fd002cb21e1ed31f0f4ae76452/github/ci/services/prometheus-stack/manifests/stack/production-e2e-workloads/prometheus-federation.yaml#L12 ?
```

Reply:
Thanks, it should be `ci.kubevirt.io/prometheus: ""`
Right, we need to expose Prometheus in the "localhost" with the 30007 port